devel/cram/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                cram
+version             0.7
+categories          devel python
+platforms           darwin
+license             GPL-2+
+maintainers         @SoapZA
```

Comment:
Please add an obfuscated version of your email and ideally `openmaintainer` as well unless there is a good reason for not doing so (but explain in a few words).